x-pack/test/plugin_functional/test_suites/resolver/index.ts

```diff
@@ -4,24 +4,174 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import expect from '@kbn/expect';
+import { WebElementWrapper } from '../../../../../test/functional/services/lib/web_element_wrapper';
+
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getPageObjects, getService }: FtrProviderContext) {
+const expectedDifference = 0.09;
+
+export default function ({
+  getPageObjects,
+  getService,
+  updateBaselines,
+}: FtrProviderContext & { updateBaselines: boolean }) {
   const pageObjects = getPageObjects(['common']);
   const testSubjects = getService('testSubjects');
+  const screenshot = getService('screenshots');
+  const find = getService('find');
+  const browser = getService('browser');
 
   describe('Resolver test app', function () {
     this.tags('ciGroup7');
 
-    beforeEach(async function () {
+    // Note: these tests are intended to run on the same page in serial.
+    before(async function () {
       await pageObjects.common.navigateToApp('resolverTest');
+      // make the window big enough that all nodes are fully in view (for screenshots)
+      await browser.setWindowSize(1920, 1200);
```

Comment:
There's another method `setScreenshotSize` which tries to take into account differences in screens scaling factors.  `setWindowsSize` also doesn't take into account the difference when running headless Chrome.

Reply:
@LeeDr Thanks for the tip. I'm trying to think through this to make sure it makes sense. Here's my thoughts:

* Resolver has a 'map' type UI. A certain portion of the map is visible at a given time. This is determined by the size of the DOM element (and any zoom level or panning changes done by the user.)
* In order to guarantee that the 3 process nodes we interact with are rendered to the DOM, the plugin needs to be rendered at a certain size (in logical pixels, as are measured using `getBoundingClientRect`.)
* I think we need to guarantee that the logical pixel size of the screen is at least big enough to show the nodes, regardless of the scaling factor.

Does this make sense? Maybe I could just use `setScreenshotSize` and give it a number that is very large (e.g. 2x as big as is needed.)